### PR TITLE
FE: Notifications Center for when a user funds your project #62

### PR DESCRIPTION
**Pull request by tsyaeger**
-[x] create a notifications model for funding events
-[x] showing notifications in the navbar as a new field

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "jsonwebtoken": "^8.5.1",
+    "material-ui-icons": "^1.0.0-beta.36",
```
Comment (evankp) [inline client/package.json]: We have @material-ui/icons already. What did you need the beta?

File: client/src/components/payments/checkoutForm.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+          return investment
+        });
+      });
+    } else {
+      console.log("Stripe.js hasn't loaded yet.");
+    }
+  };
+  
+  formatDollars = (num) => {
+    const s = ""+num
```
Comment (evankp) [inline client/src/components/payments/checkoutForm.js:123]: Can't this just be s = num or even s= `${num}`

File: client/src/components/payments/checkoutForm.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    const { userId, projectId, projectName, classes } = this.props
+    const { investmentAmount } = this.state
+    const investInDollars = this.formatDollars(investmentAmount)
+    
+    return (
+      <div>
+      <Card className={classes.card}>
+      <CardContent className={classes.content}>
+      <Typography
+      className={"MuiTypography--heading"}
```
Comment (evankp) [inline client/src/components/payments/checkoutForm.js:138]: If you want to customize this I would suggest creating a class and then using the regular method of applying styes, styles from MuiTypography should be applied default?

File: client/src/components/payments/checkoutForm.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      
+      <form onSubmit={this.handleSubmit}>
+      <CardContent className={classes.content}>
+      <CardElement
+      {...createOptions()}
+      />
+      </CardContent>
+
+      <CardContent className={classes.content}>
+        <button style={buttonStyle}>invest</button>
```
Comment (evankp) [inline client/src/components/payments/checkoutForm.js:168]: Maybe use a MaterialUi button here

File: client/src/pages/Checkout.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+        return <h3>success</h3>
+      }
+      else if (status === 'PENDING') {
+        return <h3>loading</h3>
+      }
+      else if (status !== 'FAILURE') {
+        return <h3>that didn't work. try again?</h3>//TODO: try again button
+      }
+    }
+  }
```
Comment (evankp) [inline client/src/pages/Checkout.js:40]: could probably use a switch statement here

File: client/package.json (hunk 1 above)
Comment (tsyaeger) [reply client/package.json]: yeah -- I actually uninstalled it but it didn't make it through the commit. This one fixed a bug I had, which was caused by not importing it w the @ symbol. This one allowed me to do that, but I figured it out and removed it.

File: client/src/pages/Checkout.js [hunk 6]
```diff
@@ -0,0 +1,75 @@
+import React, { Component } from 'react';
```
Comment (skassam21) [inline client/src/pages/Checkout.js:1]: This looks like it's from the other PR, probably want to make sure you branch from dev as well when starting a new feature!

File: client/src/components/payments/checkoutForm.js (hunk 2 above)
Comment (tsyaeger) [reply client/src/components/payments/checkoutForm.js]: I needed to turn it into a string before performing the replace

File: client/src/components/payments/checkoutForm.js (hunk 3 above)
Comment (tsyaeger) [reply client/src/components/payments/checkoutForm.js]: Could you expand on that? Not sure exactly what you mean.

File: client/src/components/payments/checkoutForm.js (hunk 2 above)
Comment (evankp) [reply client/src/components/payments/checkoutForm.js:123]: ah that can be done with string interpolation as well IE ``const s = `${num}` ``

File: client/src/components/payments/checkoutForm.js (hunk 3 above)
Comment (evankp) [reply client/src/components/payments/checkoutForm.js:138]: Basically, I think MuiTypography--heading is unneeded as it should be stylized through material-ui, or if you want to customize it do something like className={classes.header} or className="custom-header"

**Pull request closed without merging**